file/file.go

```diff
@@ -87,9 +98,18 @@ func (f *file) Fetch(key string) (string, error) {
 		return "", err
 	}
 
+	if f.isExpired(content) {
+		_ = f.Delete(key)
+		return "", cachego.ErrCacheExpired
+	}
+
 	return content.Data, nil
 }
 
+func (f *file) isExpired(content *fileContent) bool {
```

Comment:
![image](https://user-images.githubusercontent.com/3785409/236676512-f31528c9-224e-4c85-aaa1-f9e56a59a069.png)

Because add lock in file cache, that's what we need to do. 😁

Reply:
Ah! Okay, now I understand, was necessary to spit because of the locking, otherwise the `RUnlock` defer will be called manually.